src/patternfly/components/AppLauncher/examples/AppLauncher.md

```diff
@@ -197,6 +197,112 @@ import './AppLauncher.css'
 {{/app-launcher}}
 ```
 
+```hbs title=Favorites
+{{#> app-launcher id="app-launcher-sections-dividers-icons-and-external-links" app-launcher--IsExpanded="true" app-launcher--IsGrouped="true"}}
+  {{#> app-launcher-menu}}
+    {{#> app-launcher-menu-input}}
+      {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="search" placeholder="Filter by name..." id="' id '-text-input" name="textInput1"')}}
+        {{/form-control}}
+    {{/app-launcher-menu-input}}
+    {{#> app-launcher-group}}
+      {{#> app-launcher-group-title}}
+        Favorites
+      {{/app-launcher-group-title}}
+      <ul>
+        {{#> app-launcher-menu-item app-launcher-menu-item--type="li" app-launcher-menu-item--modifier="pf-m-wrapper pf-m-external pf-m-favorite" app-launcher-menu-item--attribute='href="#"'}}
+          {{#> app-launcher-menu-item app-launcher-menu-item--modifier="pf-m-link" app-launcher-menu-item-favorite-link--attribute='href="#" target="_blank"'}}
+            {{#> app-launcher-menu-item-icon}}
+              <img src="/assets/images/pf-logo-small.svg" alt="">
```

Comment:
are we supposed to have html here or should this be a .hbs partial?

Reply:
I leave it as HTML because it allows you to pass a unique image as the `@partial-block` to the partial.

Reply:
🔥